### PR TITLE
Refactor/display functions

### PR DESCRIPTION
**Pull request by irmakerdem**
What does this PR do?
- Refactors the horrible 😭 displaySleepInfo function

Where should your reviewer start?
- Line 91 of scripts.js

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- See first answer 🤓

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -86,47 +86,58 @@ function getUserName() {
...
+  let qualityArr = sleepId.map(daySleep => daySleep.sleepQuality);
   let averageQualityDom = globalSleep.acquireAverageMetricPerDay(qualityArr);
-  console.log('avgquality', averageQualityDom)
   averageQuality.innerText += ` ${averageQualityDom}`;
-  let lastIndexNum = [sleepId.length - 1];
-  let indexOfBigData = (sleepId[lastIndexNum]);
-  let finalIndexDate = indexOfBigData.date;
+}
+
+function specificDayStuff(sleepId, finalIndexDate) {
```
Comment (catlynbowles) [inline src/scripts.js:117]: We should rename this maybe? 👽👹

File: src/scripts.js [hunk 2]
```diff
@@ -159,12 +169,11 @@ function displayIdCardInfo(newUser) {
   strideLengthText.innerText += ` ${newUser.strideLength}`;
   dailyStepGoalText.innerText += ` ${newUser.dailyStepGoal}`;
   var userFriend = newUser.friends.map(friend => {
-    return friend = globalUserRepository.getUserDataBasedOnId(friend)
-  });
-  let userFriendNames = userFriend.map(friend => {
-    return friend.returnUserFirstName()
+    let foo = globalUserRepository.getUserDataBasedOnId(friend);
```
Comment (catlynbowles) [inline src/scripts.js:172]: I love this cleaner refactor of the friends function 🐳🏁

File: src/scripts.js [hunk 3]
```diff
@@ -86,47 +86,58 @@ function getUserName() {
...
-    return daySleep.hoursSlept
-  })
+  let finalIndexDate = dateEdit(sleepId);
+  displayAveHours(sleepId);
+  displayAveQuality(sleepId);
+  specificDayStuff(sleepId, finalIndexDate);
+  weekStuff(sleepId, finalIndexDate);
+}
 
+function dateEdit(sleepId) {
```
Comment (catlynbowles) [inline src/scripts.js:99]: This looks sooo much dryer following SRP. Great work, CIA! 🐩🌺

**Review by catlynbowles — APPROVED** [review]

**Review by ASands17 — COMMENTED** [review]
Nice work breaking this function down! 💃🏼 This is way easier to understand.

File: src/scripts.js (hunk 2 above)
Comment (ASands17) [reply src/scripts.js:172]: This looks great! We'll rename those variables once we're going through our names. 🤩

**Pull request merged**